python/foglamp/plugins/south/common/sensortag_cc2650.py

```diff
@@ -136,24 +136,31 @@ class SensorTagCC2650(object):
     is_connected = False
 
     def __init__(self, bluetooth_adr, timeout):
-        try:
-            self.bluetooth_adr = bluetooth_adr
-
-            self.con = pexpect.spawn('gatttool -b ' + bluetooth_adr + ' --interactive')
-            self.con.expect('\[LE\]>', timeout=int(timeout))
-
-            msg_debug = 'SensorTagCC2650 {} Connecting... If nothing happens, please press the power button.'.\
-                        format(self.bluetooth_adr)
-            print(msg_debug)
-            _LOGGER.debug(msg_debug)
+        attempt_count = 1
+        while attempt_count <= 5:
+            try:
+                self.bluetooth_adr = bluetooth_adr
+
+                self.con = pexpect.spawn('gatttool -b ' + bluetooth_adr + ' --interactive')
+                self.con.expect('\[LE\]>', timeout=int(timeout))
+
+                msg_debug = 'SensorTagCC2650 {} Connecting... If nothing happens, please press the power button. Attempt no. {}'.\
+                            format(self.bluetooth_adr, attempt_count)
+                print(msg_debug)
```

Comment:
Can we loose the prints please?

Reply:
This message appears only when we start server vide "python3 -m foglamp.services.core".